### PR TITLE
[Compute RN] Add 22.06 (Kepler) Update 1 release notes

### PR DESCRIPTION
**Pull request by divyabhushan-okta**

### CODE REVIEW

File: compute/rn/release-information/release-notes-22-06-update1.adoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
+== 22.06 Update 1 Release Notes
+
+The following table outlines the release particulars:
```
Comment (rcaballeromx) [inline compute/rn/release-information/release-notes-22-06-update1.adoc]: Consider changing it to:
```suggestion
The following table provides the release details.
```

File: compute/rn/release-information/release-notes-22-06-update1.adoc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+[cols="1,4"]
+|===
+|Build
+|22.06.197
+
+|Code name
```
Comment (rcaballeromx) [inline compute/rn/release-information/release-notes-22-06-update1.adoc]:
```suggestion
|Codename
```

File: compute/rn/release-information/release-notes-22-06-update1.adoc [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+|SHA-256 digest
+|5aa618314e176d03e559e58d2eba50959365cdc145cba99f5d47d90737d233bf
+|===
+
+// Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+//
+// LINK
+
+=== Improvements, fixes, and performance enhancements
```
Comment (rcaballeromx) [inline compute/rn/release-information/release-notes-22-06-update1.adoc]: Use title case for consistency
```suggestion
=== Improvements, Fixes, and Performance Enhancements
```

File: compute/rn/release-information/release-notes-22-06-update1.adoc [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+// #39668 #PCSUP-9482
+* Fixed an issue wherein the Defenders blocked application deployments on SELinux due to incorrect SELinux labeling on proxy `runc`. The issue was fixed by applying the original runc's SELinux label to the created `runc` proxy binary.
+// #39821 On-prem only
+* Fixed https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/configure/certificates[Self-signed certificate] validity period error. The value is extended and should be a whole number greater than or equal to 1.
+// #39434 PCSUP-9587 + 1749
+* Fixed an error in evaluating the custom checks for registry with CRI-O Defender. Custom checks are now run on the materialized image path.
+// #39351 PCSUP-9555
+* Fixed error that overwrote the communication port after upgrading a Defender with a custom port from the Prisma Cloud Console UI.
+
+=== Known issues
```
Comment (rcaballeromx) [inline compute/rn/release-information/release-notes-22-06-update1.adoc]:
```suggestion
=== Known Issues
```

File: compute/rn/release-information/release-notes-22-06-update1.adoc [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+// #
+
+=== End of Support Notifications
+
+// #40122
+* Debian 9 (Stretch) has reached End of Life (EOL), and users of Debian 9 will stop receiving any CVE security vulnerabilities from the https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-intelligence-stream-notifications/notifications/intelligence-stream-significant-impact[IS] feed associated with this OS version.
+
+=== Supported host Operating systems
```
Comment (rcaballeromx) [inline compute/rn/release-information/release-notes-22-06-update1.adoc]:
```suggestion
=== Supported Host Operating Wystems
```

File: compute/rn/release-information/release-notes-22-06-update1.adoc [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+=== Improvements, fixes, and performance enhancements
+//GithubIssue PCSUP-issue
+// #39791 #PCSUP-10025 On-prem
+* Fixed the broken pipe error when https://prisma.pan.dev/api/cloud/cwpp/images#operation/get-images-download[downloading a large image CSV]. The error was fixed by extending the HTTP client timeout value.
+// #40032 #PCSUP-10097 On-prem only
+* Fixed the welcome tour screen for new users.
+// #39668 #PCSUP-9482
+* Fixed an issue wherein the Defenders blocked application deployments on SELinux due to incorrect SELinux labeling on proxy `runc`. The issue was fixed by applying the original runc's SELinux label to the created `runc` proxy binary.
+// #39821 On-prem only
+* Fixed https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/configure/certificates[Self-signed certificate] validity period error. The value is extended and should be a whole number greater than or equal to 1.
```
Comment (rcaballeromx) [inline compute/rn/release-information/release-notes-22-06-update1.adoc]:
```suggestion
* Fixed the validity period error of https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/configure/certificates[self-signed certificates]. The value is extended and should be a whole number greater than or equal to 1.
```

File: compute/rn/release-information/release-notes-22-06-update1.adoc [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+* Fixed error that overwrote the communication port after upgrading a Defender with a custom port from the Prisma Cloud Console UI.
+
+=== Known issues
+
+// #
+
+=== End of Support Notifications
+
+// #40122
+* Debian 9 (Stretch) has reached End of Life (EOL), and users of Debian 9 will stop receiving any CVE security vulnerabilities from the https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-intelligence-stream-notifications/notifications/intelligence-stream-significant-impact[IS] feed associated with this OS version.
```
Comment (rcaballeromx) [inline compute/rn/release-information/release-notes-22-06-update1.adoc]:
```suggestion
* Debian 9 (Stretch) has reached End of Life (EOL), and users of Debian 9 no longer receive any CVE security vulnerabilities from the https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-intelligence-stream-notifications/notifications/intelligence-stream-significant-impact[intelligence stream] feed associated with this OS version.
```

**Review by rcaballeromx — COMMENTED** [review]
Great work and thank you! 
I just left a few suggestions with minor edits.
I added an extra item to the fixes list as well.

File: compute/admin_guide/configure/certificates.adoc [hunk 8]
```diff
@@ -11,7 +11,7 @@ NOTE: Customizing certificates is only allowed for Prisma Cloud Compute edition.
 
 |Console TLS communication
 |Console Web and API certificate
-|Web browser, API, and twistcli access to console
+|Web browser, API and twistcli access to console
```
Comment (Pubs-MV) [inline compute/admin_guide/configure/certificates.adoc]: Twistcli
Console

File: compute/admin_guide/configure/certificates.adoc [hunk 9]
```diff
@@ -300,8 +301,8 @@ Thereafter, these Defenders won't be able to establish a connection to Console u
 NOTE: Single Defenders upgraded from the Console UI don't get newly rotated certificates.
 To set up single Defenders with the new certificate, you must manually redeploy them.
 
-To identify which Defenders require redeployment, go to *Manage > Defenders > Manage > Defenders*.
-Using the *Status* column, you will be able to identify the Defenders that are using an old certificate.
```
Comment (Pubs-MV) [inline compute/admin_guide/configure/certificates.adoc]: Use the ...column to identify...

File: compute/is_notifications/notifications/intelligence-stream-significant-impact.adoc [hunk 10]
```diff
@@ -2,15 +2,14 @@
...
+The IS is constantly updated and improved to include the latest changes in security vulnerabilties, and new vulnerabilities are frequently added when detected by the Palo Alto Networks threat research teams and official data from third party feeds such as NVD. All such updates are not captured in this document.
 
-The items in this list are intended to keep you informed of only the last 7 breaking changes or changes with significant impact, and their items will be rolled out after they have gone into effect.
+The items in this list are intended to keep you informed of only the last 7 breaking changes or changes with significant impact, and it items will be rolled out after they have gone into effect.
 
 [width="90%",cols="1,1,2",options="header"]
 |=========================================================
 |Date Announced |Date Taking Effect |Description of Change
 
-| 22 July, 2022 | 31 July, 2022 | Debian 9 (Stretch) has reached End of Life (EOL), and users of Debian 9 will stop receiving any CVE security vulnerabilities associated with this OS version.
```
Comment (Pubs-MV) [inline compute/is_notifications/notifications/intelligence-stream-significant-impact.adoc]: will you remove this? the change is already live and the  phrasing is different?

File: compute/admin_guide/configure/certificates.adoc [hunk 11]
```diff
@@ -300,8 +301,8 @@ Thereafter, these Defenders won't be able to establish a connection to Console u
...
+To identify which Defenders require redployment, go to *Manage > Defenders > Manage > Defenders*.
+Using the *Status* column, you will be able identify the Defenders that are using an old certificate.
```
Comment (Pubs-MV) [inline compute/admin_guide/configure/certificates.adoc]: Use the ...column to identify...

File: compute/rn/release-information/release-notes-22-06-update1.adoc [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+
+=== Supported host Operating systems
+
+A comprehensive list of supported operating systems is found in our https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/install/system_requirements[system requirements].
```
Comment (Pubs-MV) [inline compute/rn/release-information/release-notes-22-06-update1.adoc]: For a comprehensive list of supported operating systems, refer to the https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/install/system_requirements[system requirements].

File: compute/rn/release-information/release-notes-22-06-update1.adoc [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+// Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+//
+// LINK
+
+=== Improvements, Fixes, and Performance Enhancements
+//GithubIssue PCSUP-issue
+// GH#39791 #PCSUP-10025 On-prem
+* Fixed the broken pipe error when https://prisma.pan.dev/api/cloud/cwpp/images#operation/get-images-download[downloading a large image CSV]. The error was fixed by extending the HTTP client timeout value.
+// GH#40032 #PCSUP-10097 On-prem only
+* Fixed the welcome tour screen for new users.
```
Comment (grevach) [inline compute/rn/release-information/release-notes-22-06-update1.adoc]: @Pubs-MV this fix is for non-admin new users

File: compute/rn/release-information/release-notes-22-06-update1.adoc [hunk 14]
```diff
@@ -0,0 +1,55 @@
...
+// GH#39668 #PCSUP-9482
+* Fixed an issue wherein the Defenders blocked application deployments on SELinux due to incorrect SELinux labeling on proxy `runc`. The issue was fixed by applying the original runc's SELinux label to the created `runc` proxy binary.
+// GH#39821 On-prem only
+* Fixed the validity period error of https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/configure/certificates[self-signed certificates]. The value is extended and should be a whole number greater than or equal to 1.
```
Comment (grevach) [inline compute/rn/release-information/release-notes-22-06-update1.adoc]: @Pubs-MV maybe we should mention that the limit was 365 and now it is unlimited

Comment (divyabhushan-okta) [reply compute/rn/release-information/release-notes-22-06-update1.adoc]: @grevach How about - "The limit of 365 has been waved off and the value can now be a whole number greater than or equal to 1"?

File: compute/rn/release-information/release-notes-22-06-update1.adoc [hunk 15]
```diff
@@ -0,0 +1,53 @@
...
+|===
+
+// Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+//
+// LINK
+
+=== Improvements, fixes, and performance enhancements
+//GithubIssue PCSUP-issue
+// #39791 #PCSUP-10025 On-prem
+* Fixed the broken pipe error when https://prisma.pan.dev/api/cloud/cwpp/images#operation/get-images-download[downloading a large image CSV]. The error was fixed by extending the HTTP client timeout value.
```
Comment (MorShabiPAN) [inline compute/rn/release-information/release-notes-22-06-update1.adoc]: @Pubs-MV we need to specify that this error occurred only for secondary consoles when using Projects. This is relevant for Self-hosted deployments with projects.

File: compute/rn/release-information/release-notes-22-06-update1.adoc [hunk 16]
```diff
@@ -0,0 +1,55 @@
...
+
+|SHA-256 digest
+|5aa618314e176d03e559e58d2eba50959365cdc145cba99f5d47d90737d233bf
+|===
+
+// Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+//
+// LINK
+
+=== Improvements, Fixes, and Performance Enhancements
```
Comment (MorShabiPAN) [inline compute/rn/release-information/release-notes-22-06-update1.adoc:27]: @Pubs-MV where do you think is the right place to mention new supported orchestrations like TKGI and [GKE arm](https://spring.paloaltonetworks.com/twistlock/twistlock/issues/39581)?

cc @grevach

File: compute/rn/release-information/release-notes-22-06-update1.adoc (hunk 14 above)
Comment (grevach) [reply compute/rn/release-information/release-notes-22-06-update1.adoc]: @divyabhushan sounds good

**Comment by MayaShani** [comment]
@rcaballeromx - please see my comment on https://spring.paloaltonetworks.com/twistlock/twistlock/issues/39751

**Comment by MayaShani** [comment]
Please see my comment on https://spring.paloaltonetworks.com/twistlock/twistlock/issues/39434

**Comment by MayaShani** [comment]
Please see my comment on https://spring.paloaltonetworks.com/twistlock/twistlock/issues/38819

**Comment by MayaShani** [comment]
Please see my comment on https://spring.paloaltonetworks.com/twistlock/twistlock/issues/37579

**Comment by MayaShani** [comment]
Also - what about the following ones? where is the updated system requirement doc PR ?
https://spring.paloaltonetworks.com/twistlock/twistlock/issues/39685
https://spring.paloaltonetworks.com/twistlock/twistlock/issues/39316
https://spring.paloaltonetworks.com/twistlock/twistlock/issues/39261

File: compute/rn/release-information/release-notes-22-06-update1.adoc [hunk 17]
```diff
@@ -0,0 +1,64 @@
...
+// GH#39434 PCSUP-9587 + 1749
+* Fixed an error in evaluating the custom compliance checks for registry with CRI-O (Dockerless) Defender. Custom checks are now run on the materialized image path.
+// GH#39351 PCSUP-9555
+* Fixed error that overwrote the communication port after upgrading a Defender with a custom port from the Prisma Cloud Console UI.
+// GH#37579 PCSUP-8519
+* Fixed an issue that showed different fixes for the same CVE on a single image. Each CVE vulnerability is consolidated and grouped according to OS version for each image and package.
+// GH#38819 PCSUP-9069
+* Fixed the filter to list all the tags when multiple images have the same digest.
+//GH#39581
+* Added support for GKE version 1.24.1 running on ARM64 architecture.
```
Comment (MorShabiPAN) [inline compute/rn/release-information/release-notes-22-06-update1.adoc]: @Pubs-MV I suggest emphasizing the new features and supported orchestrators on top of the bug fixes, maybe by moving them to the head of the list or listing them under another topic. 
Specifically, regarding the GKE ARM support, we can also add a link to the [Blogpost](https://www.paloaltonetworks.com/blog/prisma-cloud/supports-arm-workloads-on-google-cloud-and-gke/#:~:text=Prisma%20Cloud%20Supports%20Arm%20Workloads%20on%20Google%20Cloud%20and%20GKE,-Cloud%20Workload%20Protection&text=Prisma%20Cloud%20integrates%20with%20GKE,Google%20Kubernetes%20Engine%20(GKE).)

File: compute/rn/release-information/release-notes-22-06-update1.adoc [hunk 18]
```diff
@@ -0,0 +1,64 @@
...
+//GH#39938
+* Added support for VMware Tanzu Kubernetes Grid Integrated (TKGI) version 1.14
+//GH#39977
+* Added support for VMware Tanzu Kubernetes Grid Multicloud (TKGM) version 1.5.1 on Photon 3 and Ubuntu 20.04.03 LTS
+
+=== Known Issues
+
+// GH#39682 PCSUP-9275
+* Defenders are not accepting the self-signed proxy certificate configured for TLS intercept proxies.
```
Comment (grevach) [inline compute/rn/release-information/release-notes-22-06-update1.adoc]: @rcaballeromx this is not a known issue, we have fixed the bug.

File: compute/rn/release-information/release-notes-22-06-update1.adoc (hunk 17 above)
Comment (Pubs-MV) [reply compute/rn/release-information/release-notes-22-06-update1.adoc]: yes

File: compute/rn/release-information/release-notes-22-06-update1.adoc (hunk 18 above)
Comment (Pubs-MV) [reply compute/rn/release-information/release-notes-22-06-update1.adoc]: Gal, we can add the workaround this and say how to do it using an unsupported API: https://spring.paloaltonetworks.com/twistlock/twistlock/issues/39082

File: compute/rn/release-information/release-notes-22-06-update1.adoc (hunk 16 above)
Comment (Pubs-MV) [reply compute/rn/release-information/release-notes-22-06-update1.adoc:27]: we added a new bullet to the top so it is the highlight for the update @MorShabiPAN

**Review by iansk — APPROVED** [review]

**Pull request merged**